src/1-Profile/Edit-Profile.tsx

```diff
@@ -158,11 +173,25 @@ const EditProfile = ({navigation}:StackNavigationProps):JSX.Element => {
             <Text style={styles.header}>Edit Profile</Text>
```

Comment:
Can we display the profile image in a circle at the top of the page?  Maybe pencil icon to indicate clicking  on picture opens image edit page.  Show default image if profile image does not exist yet.